### DISCUSSION TITLE
White list websites from user-agent

### DISCUSSION THREAD

**Discussion by byjrg** [Floorp-Projects/Floorp#670] [Feature Request]
Well, as the title says, it would be nice to have the ability to whitelist some websites that doesn't work properly while using a different user agent (for example you can't write on Google Docs or maps load super slow on Google Maps).

**Comment by breadbeforerice** [comment]
I need to temporarily disable impersonation as it blocks me when I log in.

**Comment by AxeldeWater** [comment]
Came here to request the same: use a different user agent for a couple sites such as youtube and then use the default user agent for everything else.
